bot/bot.py

```diff
@@ -1,628 +1,34 @@
-from telegram import Update
-from telegram import ParseMode
+from typing import NoReturn
+
 from telegram.ext import (
-    CallbackContext,
     CommandHandler,
     Filters,
     MessageHandler,
     Updater,
-    CallbackQueryHandler,
 )
-from typing import NoReturn
 
+import handlers
 from app.splitwise import SplitwiseApp
 
-import buttons
-
 
 class TelegramBot:
     def __init__(
         self,
         token: str,
+        db_name: str = 'database.sqlite',
     ):
-        self.splitwise = SplitwiseApp()
+        self.splitwise = SplitwiseApp(db_name=db_name)
         self.updater = Updater(token)
-        self.expenses = dict()
-        self.debts = dict()
-        self.states = dict()
         dispatcher = self.updater.dispatcher
-        dispatcher.add_handler(CommandHandler("create_event", self.create_event_handler))
-        dispatcher.add_handler(CommandHandler("add_expense", self.add_expense_handler))
-        dispatcher.add_handler(CommandHandler("show_debts", self.show_debts_handler))
-        dispatcher.add_handler(CommandHandler("users_of_event", self.users_of_event_handler))
-        dispatcher.add_handler(CommandHandler("join_event", self.join_event_handler))
-        dispatcher.add_handler(CommandHandler("start", self.start_handler))
-        dispatcher.add_handler(CallbackQueryHandler(self.callback_query_handler))
-        dispatcher.add_handler(MessageHandler(Filters.text, self.text_handler))
+        dispatcher.add_handler(handlers.MenuButtonsConversationHandler().get_conversation_handler())
```

Comment:
Does order of adding handlers matter here?

Reply:
https://python-telegram-bot.readthedocs.io/en/stable/telegram.ext.dispatcher.html#telegram.ext.Dispatcher.add_handler
TL;DR: Order and priority counts

Reply:
Order is right, nothing to fix